.helm/.env-example

```diff
@@ -8,3 +8,19 @@ POSTGRES_ADMIN_URL=
 BASIC_AUTH_ENABLED=false
 BASIC_AUTH_USERNAME=test
 BASIC_AUTH_PASSWORD=test
+
+BACKUP_SCHEDULE=@hourly
```

Comment:
Is that the default? Isn't that a bit much? One prod backup is around 16MB right now, but on eCamp v2 a prod dump is roughly 1GB.

Reply:
This is just if you want to deploy the containers and the helm chart from your local machine.
The problem is shell globbing, which turns your cron expression "2 * * * *" into "2 <list of files in directory> <list of files in directory>...".
But for my tests i even set it to all 3 minutes, because i don't want to wait that long for the job to execute, and i don't leave the deployment up for long.

The prod backup we used on the last hackaton ist 2.4 MB compressed.